src/components/branded-card/index.tsx

```diff
@@ -20,7 +20,10 @@ import patternVault from './img/vault.png'
 import patternWaypoint from './img/waypoint.png'
 import s from './branded-card.module.css'
 
-const PATTERN_IMG_MAP: Record<ProductSlug, StaticImageData> = {
+const PATTERN_IMG_MAP: Record<
+	Exclude<ProductSlug, 'well-architected-framework'>,
```

Comment:
@im2nguyen I'm curious about this addition. What issue did this fix?

Reply:
It didnt let me build/commit since there was a type error. If you look through the original PR, this is where the build step was failing